azure_monitor/src/azure_monitor/protocol.py

```diff
@@ -308,16 +443,16 @@ class Request(BaseObject):
 
     def __init__(
         self,
-        ver=2,
-        id="",
-        duration="",
-        response_code="",
-        success=True,
-        source=None,
-        name=None,
-        url=None,
-        properties=None,
-        measurements=None,
+        ver: int = 2,
+        id: str = "",
+        duration: str = "",
+        response_code: str = "",
+        success: bool = True,
+        source: str = None,
+        name: str = None,
+        url: str = None,
+        properties: typing.Dict[str, any] = None,
+        measurements: any = None,
```

Comment:
I believe `measurements` is a dict.

https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/RequestData.bond#L46